### DISCUSSION TITLE
Welcome, Noman!

### DISCUSSION THREAD

**Discussion by NomanGhiasy** [ParanoidUser/yolo#1039] [Subscribe]
Leaving a message for Yolo Badge

**Comment by ParanoidUser** [comment]
Ahoy @NomanGhiasy, thank you for your interest in the project! To get a Yolo badge, simply follow the steps in [the guide](https://github.com/ParanoidUser/yolo/discussions/18).

**Note:** Don't forget to assign a reviewer on step 4. 🔭

**Another note:** You should receive a repo collaborator invite that allows you to merge PRs (steps 5-6).

Let me know if you have any questions. Welcome aboard! 🚢⭐

**Reply by NomanGhiasy** [reply]
I click on the accept invite and it gives 404 error. Could you send another invite please. Thank you

**Reply by opticaldrive** [reply]
Link seems too work now…

**Reply by NomanGhiasy** [reply]
@V205Arduino it worked

**Reply by opticaldrive** [reply]
great!
